tools/wpt/browser.py

```diff
@@ -1137,34 +1151,39 @@ def install_webdriver(self, dest=None, channel=None, browser_binary=None):
             self.logger.info("Successfully downloaded MSEdgeDriver to %s" % edgedriver_path)
         return find_executable(self.edgedriver_name, dest)
 
+    def install_webdriver(self, dest=None, channel=None, browser_binary=None):
+        if browser_binary is None:
+            browser_binary = self.find_binary(channel)
+        return self.install_webdriver_by_version(
```

Comment:
If `version` is ultimately unused in `install_webdriver_by_version` and that's OK, then this whole `install_webdriver` probably collapses into just an alias for `install_webdriver_by_version` and one can probably be simplified away.

Reply:
Thanks, that was a bug. Updated this function to use version correctly.

Reply:
Updated install_webdriver_by_version to use the version parameter. I'd like to keep these two function as-is to follow the established pattern in this file.